backend/app/controllers/spree/admin/search_controller.rb

```diff
@@ -1,36 +1,38 @@
 module Spree
   module Admin
     class SearchController < Spree::Admin::BaseController
+      respond_to :json
+      layout false
+
       # http://spreecommerce.com/blog/2010/11/02/json-hijacking-vulnerability/
       before_action :check_json_authenticity, only: :index
-      respond_to :json
 
       # TODO: Clean this up by moving searching out to user_class_extensions
       # And then JSON building with something like Active Model Serializers
       def users
         if params[:ids]
-          @users = Spree.user_class.where(:id => params[:ids].split(','))
+          @users = Spree.user_class.where(id: params[:ids].split(',').flatten)
         else
           @users = Spree.user_class.ransack({
-            :m => 'or',
-            :email_start => params[:q],
-            :ship_address_firstname_start => params[:q],
-            :ship_address_lastname_start => params[:q],
-            :bill_address_firstname_start => params[:q],
-            :bill_address_lastname_start => params[:q]
+            m: 'or',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.